PKHeX.Core/MysteryGifts/WB7.cs

```diff
@@ -315,13 +315,10 @@ public override string OriginalTrainerName
     /// <remarks>
     /// Gifts received can be locally traded to the international release, so there is no need to consider residence or transferability.
     /// </remarks>
-    public bool IsMainlandChinaGift => CardID is (1501);
+    public bool IsMainlandChinaGift => CardID is (1501 or 1503);
```

Comment:
Note for future-us: if they add any more events, we can likely relax this range check to

```cs
CardID is (> 1500 and <= 1550);
```
or something, just so that it doesn't need to be revised for a 4th+ card :D